File: src/src/junitjava/logic/Calculation.java
package junitjava.logic;

public class Calculation {

    public static int findMax(int arr[]){

        int max = 0;
        for(int i = 0; i < arr.length; i++){
            if(max < arr[i])
                max = arr[i];
        }

        return max;
    }

    public static long findCube(long number){

        return number * number * number;
    }

    public static String reverseString(String str){

        char arr[] = str.toCharArray();
        String s = "";
        for(int i = arr.length - 1; i >= 0; i--){
            s = s + arr[i];
        }

        return s;
    }
}

File: src/src/junitkotlin/logic/Calculation.kt
package junitkotlin.logic

class Calculation {

    companion object{

        fun findMax(arr : IntArray) : Int{

            var max = 0
            for(i in 0 until arr.size){
                if (max < arr[i])
                    max = arr[i]
            }

            return max
        }

        fun findCube(number : Long) : Long{

            return number * number * number
        }

    }
}